README.md

```diff
@@ -1 +1,157 @@
-# wordvecspace
\ No newline at end of file
+# Word Vector Space
+
+A python module that helps in loading and performing operations on word vector spaces created using Google's Word2vec tool.
+
+## Installation
+
+> Prerequisites: Python2.7
+
+```bash=!
+sudo pip install numpy pandas numba
+sudo apt install libopenblas-base
+```
+
+## Usage
+
+```python=!
+
+>>> from wordvecspace import WordVecSpace, UnknownWord, UnknownIndex
+
+# Load the data (Vector and Vocab information)
+>>> wv = WordVecSpace('/path/to/data/')
+>>> wv.load()
+
+# Check if a word exists or not in the word vector space
+>>> print wv.does_word_exist("india")
+True
+
+>>> print wv.does_word_exist("inidia")
+False
+
+# Get the index of a word
+>>> print wv.get_word_index("india", raise_exc=False)
+509
+
+>>> print wv.get_word_index("inidia", raise_exc=False)
```

Comment:
@RamanjaneyuluIdavalapati Isn't `raise_exc=False` the default? If yes, then no need to mention here.